aws/integration.yaml

```diff
@@ -74,15 +74,23 @@ Resources:
               Effect: Allow
               Resource: '*'
               Action:
-                - rds:Describe*
-                - rds:List*
-                - cloudwatch:Get*
-                - cloudwatch:List*
+                - aws-portal:ViewBilling
+                - budgets:Describe*
+                - ce:Describe*
                 - ce:Get*
                 - ce:List*
-                - ce:Describe*
-                - aws-portal:ViewBilling
+                - cloudwatch:Describe*
+                - cloudwatch:Get*
+                - cloudwatch:List*
                 - iam:SimulatePrincipalPolicy
+                - pi:Describe*
```

Comment:
What is pi?

Reply:
performance insights: https://docs.aws.amazon.com/performance-insights/latest/APIReference/Welcome.html